src/SumoCoders/FrameworkCoreBundle/Resources/doc/development/pagination.md

```diff
@@ -0,0 +1,70 @@
+# Using pagination
+
+Pagination is a nice way to handle large amounts of data over multiple pages.
+
+## Controller
+
+In the controller the following code should be used. If necessary a custom adapter can be used.
+
+```
+<?php
+use Symfony\Bundle\FrameworkBundle\Controller\Controller,
```

Comment:
not really a fan of adding usestatements like this, you can group them but just to type one use and using , seems a bit strange to me

Reply:
fixed 82caa0e